openstack/orchestration/v1/stackresources/results.go

```diff
@@ -75,18 +86,6 @@ func (r ResourcePage) IsEmpty() (bool, error) {
 	return len(resources) == 0, nil
 }
 
-// LastMarker returns the last container name in a ListResult.
```

Comment:
This change should have been in https://github.com/rackspace/gophercloud/pull/466. Apologies